File: app/src/main/java/projet/Model/Fonctionnalite.java
package projet.Model;

public class Fonctionnalite {
    private String nom;
    private String description;
    private String note;

    public Fonctionnalite(String nom, String description, String note){
        this.nom = nom;
        this.description=description;
        this.note = this.note;
    }

    public String getNom() {
        return this.nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getDescription() {
        return this.description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getNote() {
        return this.note;
    }

    public void setNote(AppManager.Card note) {
        this.note = note.toString();
    }

}

File: Assets/Scripts/Model/Fonctionnalite.cs
using UnityEngine;
using UnityEditor;
using UnityEngine.UIElements;
using UnityEditor.UIElements;
using System;
using PlanningPoker;

/// @class Fonctionnalite
/// @brief Représente une fonctionnalité à estimer
///
/// Une fonctionnalité contient un nom, une description, et une note attribuée via le processus d'estimation

namespace PlanningPoker{
    [System.Serializable]
    public class Fonctionnalite 
    {
        /// @brief Nom de la fonctionnalité
        [SerializeField]private string nom;

        /// @brief Description détaillée de la fonctionnalité
        [SerializeField]private string description;

        /// @brief Note associée à la fonctionnalité, sous forme de chaîne de caractères
        [SerializeField]private string note;

        /// @brief Getter pour le nom de la fonctionnalité
        /// @return: Le nom de la fonctionnalité
        public string getNom() 
        {
            return this.nom;
        }

        /// @brief Setter pour le nom de la fonctionnalité
        /// @param nom: Le nom à attribuer
        public void setNom(string nom) 
        {
            this.nom = nom;
        }

        /// @brief Getter pour la description de la fonctionnalité
        /// @return La description de la fonctionnalité
        public string getDescription() 
        {
            return this.description;
        }

        /// @brief Setter pour la description de la fonctionnalité
        /// @param description: La description à attribuer
        public void setDescription(string description) 
        {
            this.description = description;
        }

        /// @brief Getter pour la note associée à la fonctionnalité
        /// @return: La note sous forme de chaîne de caractères, ou null si aucune note n'a été attribuée
        public string getNote() 
        {
            return this.note;
        }

        /// @brief Setter pour la note associée à la fonctionnalité
        /// @param note: La note à attribuer (instance de Card)
        public void setNote(Card note) 
        {
            this.note = note != null ? note.toString() : null;
        }

    }

    [System.Serializable]
    public class FonctionnaliteWrapper
    {
        public Fonctionnalite[] fonctionnalites;
    }
}
